komga-webui/src/locales/en.json

```diff
@@ -51,24 +51,24 @@
     "button_scan": "Scan",
     "button_select_series": "Select Series",
     "field_import_path": "Import from folder",
-    "info_part1": "This screen lets you import files that are outside your existing libraries. You can only import files into existing Series, in which case Komga will move or copy the files into the directory of the chosen Series.",
-    "info_part2": "If you choose a number for a book, and a book already exists with that number, then you will be able to compare the 2 books. If you decide to import the book, Komga will upgrade the existing book with the new one, effectively replacing the old file with the new.",
```

Comment:
Much shorter. Wrong tense. Lots of invoked concepts needlessly.
"2 books" is an error. 1-10 should be typed out. Self-referencing (Komga).

Reply:
What do you mean by "wrong tense" ?

Reply:
It isn't so much that, or strictly wrong as it is a garden-path sentence.
"compare the 2 books" is what we are trying to get at.

Reply:
> garden-path sentence

what does that mean?

Reply:
The nomenclature from what we can tell, and taxing, according to sources is something that happens to while appearing trivial, only reveals the meaning, as something found at the end of the sentence, for that reson.

Reply:
Thanks for the explanation.